package.json

```diff
@@ -16,22 +16,29 @@
     "amplitude-js": "^8.3.1",
     "antd": "^4.16.6",
     "awesome-react-icons": "^1.4.8",
+    "aws-sdk": "^2.951.0",
     "bootstrap": "^4.6.0",
     "evergreen-ui": "^6.1.0",
     "express": "^4.17.1",
     "http-proxy-middleware": "^2.0.1",
     "js-cookie": "^2.2.1",
+    "material-design-icons": "^3.0.1",
     "material-ui-search-bar": "^1.0.0",
     "next": "11.0.1",
+    "next-s3-upload": "^0.1.5",
     "next-sitemap": "^1.6.140",
     "pure-react-carousel": "^1.27.6",
     "react": "17.0.2",
     "react-bootstrap": "^1.6.1",
     "react-dom": "17.0.2",
+    "react-dropzone": "^11.3.4",
+    "react-icons": "^4.2.0",
```

Comment:
i think material-ui already has icons we can use. not sure we should install another library